packages/workbox-build/src/lib/utils/filter-files.js

```diff
@@ -71,7 +70,7 @@ module.exports = (fileDetails, options) => {
   // {url, revision} objects, with path.sep replaced with /.
   const normalizedManifest = filteredFileDetails.map((fileDetails) => {
     return {
-      url: fileDetails.file.replace(path.sep, '/'),
+      url: fileDetails.file.replace(/\\/g, '/'),
```

Comment:
Thanks for the PR, @XhmikosR! In theory this should be  addressing the issue but AppVeyor appears to currently be timing out. We'll need to wait on that before moving ahead.

@gauntface I tried to login to restart the build but don't appear to be on the AV for Workbox. Can has privs?